packages/neos-ui/src/Containers/LeftSideBar/NodeTree/index.js

```diff
@@ -123,6 +146,14 @@ export default class NodeTree extends PureComponent {
 
         return (
             <Tree className={classNames}>
+                <a
+                    role="button"
```

Comment:
```suggestion
                <button
```

This will probably require some more reset work on the CSS side, but its better to let this be an actual `<button/>`.

Reply:
Adjusted! (Including updated css)